src/components/date-picker/DatePicker.vue

```diff
@@ -41,24 +41,25 @@
             filled
             @click:clear="emitClear()"
             @keydown="$event.preventDefault()"
-            @keyup.enter="emitDate(dateText)"
+            @keyup.enter="emitDate(date)"
           />
         </span>
       </template>
       <v-date-picker
         id="date-picker-calendar"
-        v-model="dateText"
```

Comment:
The Vuetify 3 DatePicker does not seem to be working with `string` dates, so I switched the private `dateText` variable to be `data` with type `Date`.